### DISCUSSION TITLE
Multilingual full text search?

### DISCUSSION THREAD

**Discussion by sebnun** [appwrite/appwrite#8811] [Q&A]
Hi, I'm currently using Supabase, I'm evaluating migrating to Appwrite.
On Supabase I can use pgroonga for multilingual full text search, is there something equivalent in Appwrite?

I see in the docs that full text search is available, but I'm not seeing what languages are supported.
In particular, I'm looking for support for languages not separated by white space like Chinese, Japanese, Thai, etc.

**Comment by DH-555** [comment]
I think that currently there's not a direct way to make multilingual searches. However you can use appwrite functions to translate them to the original language and then perform the search.

An alternative could be making an integration with something like Algolia:

https://appwrite.io/integrations/search-algolia

https://www.algolia.com/doc/guides/managing-results/optimize-search-results/handling-natural-languages-nlp/how-to/multilingual-search/
